### PR TITLE
[CHORE] Templates clean-up

### PR DESCRIPTION
**Pull request by TiBiBa**
**Description**
In this PR we continue with our ongoing journey of cleaning-up all templates and improving responsiveness. The biggest changes are made to all "shadow boxes" such as login, reset password and signup. These are currently fixed at a x percentage of the view width. However, it would be nice to make this dependent on the size itself: which we do in this PR. 

**Fixes**
No direct issue, we again work towards #2776.

**How to test**
<To-Do>

### CODE REVIEW

**Review by Felienne — APPROVED** [review]
Seems to not to break anything!

**Comment by TiBiBa** [comment]
My bad, this should still be a draft.

**Comment by TiBiBa** [comment]
After fixing the learn-more page (today), this PR should also be ready for merge. It might be a bit unclear on the exact added value, but we simplify a lot of styling to be more future-proof and better readable by other contributors. Other than that all "boxes" have been made responsive.

**Comment by TiBiBa** [comment]
Note to self, we have to fix the following two boxes:
- learn-more
- signup

**Comment by TiBiBa** [comment]
Example of the signup responsive re-styling:
**Old**
Schermafbeelding 2022-07-07 om 13 12 13

**New**
Schermafbeelding 2022-07-07 om 13 12 43

**Comment by TiBiBa** [comment]
We also update the learn-more form:
**Old**
Schermafbeelding 2022-07-08 om 10 46 28

**New**
Schermafbeelding 2022-07-08 om 10 46 40

File: build-tools/heroku/tailwind/styles.css [hunk 1]
```diff
@@ -27,6 +27,7 @@ h4 {
   @apply font-bold text-base;
 }
 
+/* TB: Why do we have this? It makes it (a lot) harder to style these element within the templates... */
```
Comment (Felienne) [inline build-tools/heroku/tailwind/styles.css:30]: @TiBiBa Do we need an issue here for someone to dive into this? I also don't remember why we have this :)

Comment (Felienne) [inline build-tools/heroku/tailwind/styles.css:30]: @TiBiBa Do we need an issue so we don't forget to look into this? (I also don't remember how this came to do)

**Review by Felienne — APPROVED** [review]
Looks nicer indeed!!

**Pull request merged**